commcare_connect/opportunity/views.py

```diff
@@ -1009,3 +1010,21 @@ def apply_opportunity_invite(request, application_id, org_slug=None, pk=None):
         f"Application for the opportunity '{application.managed_opportunity.name}' has been successfully submitted.",
     )
     return redirect("opportunity:list", org_slug)
+
+
+@org_member_required
+def opportunity_user_invite(request, org_slug=None, pk=None):
+    opportunity = get_object_or_404(Opportunity, organization=request.org, id=pk)
+    form = OpportunityUserInviteForm(data=request.POST or None)
+    if form.is_valid():
```

Comment:
Is this part shared with the other invite view? May be nice to create a shared function

Reply:
Fixed. I have made some changes to make the forms inherit functionality. [516e6c9](https://github.com/dimagi/commcare-connect/pull/380/commits/516e6c9856a7770c295c87c89f987b3e5887e906)